### DISCUSSION TITLE
Amount of "digital" and "analog" pictures

### DISCUSSION THREAD

**Discussion by DieterPN** [jomjol/AI-on-the-edge-device#720] [Q&A] [answered]
Very interesting project!

Could dedicated *.tflite files for my own Water- and Gas-Meter improve the recognition rate? 

Which amount of "digital" and "analog" training pictures are roughly required if I would create *.tflite files by myself.

Best regards

Dieter

**Comment by jomjol** [comment, marked as answer]
Yes, they can. You need about 2-3 perfectly aligned image per digit. And about 100 images for the analog pointer (0.0, 0.1, 0.2, ....9.8, 9.9). It is not a problem, if some of them are missing.

**Comment by DieterPN** [comment]
I have understood log-files are stored on the SD-Card. Is there an option to read one directory or all log-files at once over WLAN connection?

**Comment by jomjol** [comment]
No this is not possible over the fileserver.
